### PR TITLE
INJI-29 | 31 - iOS automation workflow and Android PR workflow

### PR DESCRIPTION
**Pull request by adityankannan-tw**
[iOS] https://mosip.atlassian.net/browse/INJI-29

- Created a pipeline for iOS build creation.
- Configured Fastlane for iOS.
- Github workflow UI for theme, url, testers group, branch.

[Android] https://mosip.atlassian.net/browse/INJI-31

- Created a PR trigger for Android build.
- Configured Fastlane for Android.

### CODE REVIEW

File: .github/workflows/android-build-verify.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 name: ID PASS - MOSIP Resident Application
 
+env:
+  backendServiceDefaultUrl: https://api.qa-121.mosip.net
```
Comment (ckm007) [inline .github/workflows/android-build-verify.yml:4]: can we make the URL as mentioned below to make the same generic.
https://api.sandbox.mosip.net

File: .github/workflows/android-build-verify.yml [hunk 2]
```diff
@@ -43,33 +52,58 @@ jobs:
         run: |
           echo "${{ secrets.ENV_FILE }}" > .env.local > android/local.properties
 
-      - name: Setup branch and env 
+      - name: Setup branch and env
```
Comment (ckm007) [inline .github/workflows/android-build-verify.yml:55]: lets not remove the comments.

File: .github/workflows/android-custom-build.yml [hunk 3]
```diff
@@ -46,41 +45,74 @@ jobs:
...
+          DEBUG_KEYSTORE_ALIAS: androiddebugkey
+          DEBUG_KEYSTORE_PASSWORD: '${{secrets.INJI_ANDROID_DEBUG_STOREPASS}}'
+      
+      - name: Upload Artifact to Actions
         uses: actions/upload-artifact@v3.1.1
         with:
-          name: apk-output
-          path: android/app/build/outputs/apk/newlogic/release/
-          retention-days: 10
+          name: Inji-apk
```
Comment (ckm007) [inline .github/workflows/android-custom-build.yml:116]: make name as Inji only

File: .github/workflows/android-build-verify.yml (hunk 1 above)
Comment (adityankannan-tw) [reply .github/workflows/android-build-verify.yml:4]: Changes are made

File: .github/workflows/android-build-verify.yml (hunk 2 above)
Comment (adityankannan-tw) [reply .github/workflows/android-build-verify.yml:55]: added back the comments

File: .github/workflows/android-build-verify.yml [hunk 4]
```diff
@@ -43,33 +52,58 @@ jobs:
...
         run: |
-          # Strip git ref prefix from version
           echo "BRANCH_NAME=$(echo ${{ github.ref }} | sed -e 's,.*/\(.*\),\1,')" >> $GITHUB_ENV
           echo "GPG_TTY=$(tty)" >> $GITHUB_ENV   
-      - name: Setup branch and GPG public key 
+
+      - name: GPG public key
```
[the remark this answers is not in the document]
Comment (adityankannan-tw) [reply .github/workflows/android-build-verify.yml:60]: Changes are made

File: .github/workflows/android-custom-build.yml (hunk 3 above)
Comment (adityankannan-tw) [reply .github/workflows/android-custom-build.yml:116]: Renamed to Inji

File: .github/workflows/android-build-verify.yml [hunk 5]
```diff
@@ -1,5 +1,8 @@
 name: ID PASS - MOSIP Resident Application
```
Comment (ckm007) [inline .github/workflows/android-build-verify.yml:1]: Lets change the Workflow name to 
Android build verification

File: .github/workflows/android-custom-build.yml [hunk 6]
```diff
@@ -1,15 +1,15 @@
 name: ID PASS - MOSIP Resident Application Custom build
```
Comment (ckm007) [inline .github/workflows/android-custom-build.yml:1]: Also rename this to 
Android Custom Build

File: .github/workflows/android-build-verify.yml (hunk 5 above)
Comment (adityankannan-tw) [reply .github/workflows/android-build-verify.yml:1]: Done

File: .github/workflows/android-custom-build.yml (hunk 6 above)
Comment (adityankannan-tw) [reply .github/workflows/android-custom-build.yml:1]: changed

**Review by ckm007 — APPROVED** [review]

**Pull request merged**